app/controllers/api/v1/accounts/custom_filters_controller.rb

```diff
@@ -13,10 +15,13 @@ def create
       permitted_payload.merge(account_id: Current.account.id)
     )
     render json: { error: @custom_filter.errors.messages }, status: :unprocessable_entity and return unless @custom_filter.valid?
+
+    update_filter_conversation_count(@custom_filter)
   end
 
   def update
     @custom_filter.update!(permitted_payload)
+    update_filter_conversation_count(@custom_filter)
```

Comment:
This logic looks more suited for the controller than the model ( based on the pattern which we follow )
- the `update_filter_conversation_count` can happen in an after-save callback. 
-  `Events::Types` is already included in application record